src/ir/dot/dot_generator_test.cc

```diff
@@ -140,8 +142,8 @@ TEST_F(DotGeneratorTest, CreatesAttributesInNodesForOperation) {
   EXPECT_THAT(
       GetNodes(dot_graph_lines),
       testing::UnorderedElementsAre(
-          "B0 [shape=Mrecord]",
-          R"(B0_0 [label=<<TABLE BORDER="0" CELLBORDER="1" CELLSPACING="0"><TR><TD COLSPAN="1">&nbsp;</TD></TR><TR><TD COLSPAN="1">core.plus [string: something, ttl: 10] </TD></TR><TR><TD COLSPAN="1" PORT="out">out</TD></TR></TABLE>>])"));
+          "Bb0 [shape=Mrecord]",
+          R"(Bb0_%0 [label=<<TABLE BORDER="0" CELLBORDER="1" CELLSPACING="0"><TR><TD COLSPAN="1">&nbsp;</TD></TR><TR><TD COLSPAN="1">core.plus [string: something, ttl: 10] </TD></TR><TR><TD COLSPAN="1" PORT="out">out</TD></TR></TABLE>>])"));
```

Comment:
Unfortunately, the `%0` in  `Bb0_%0` will cause issues rendering the generated dot file. Can you enclose the name with quotes (`"`)?  (I have added suggestions at the relevant places.)

Reply:
Done!